.github/workflows/PR-5.x.yaml

```diff
@@ -2,6 +2,7 @@ name: PR:5.x
 
 on:
   pull_request:
+    types: [opened, reopened, labeled, unlabeled, synchronize]
```

Comment:
> labeled, unlabeled

Please don't use this. It retriggers ALL pipelines if any label is updated. It is very inconvenient.

Proposed policy is re-triggering job manually by maintainers after adding necessary label.

Reply:
Got your idea, removed this line.